Source/Renderers/DownASTRenderable.swift

```diff
@@ -23,7 +23,7 @@ public protocol DownASTRenderable: DownRenderable {
     func toAST(_ options: DownOptions) throws -> UnsafeMutablePointer<cmark_node>
 }
 
-public extension DownASTRenderable {
+extension DownASTRenderable {
```

Comment:
Since the methods in this extension are `public`, marking this as `public` too is a warning in the Swift 5 compiler.